app/src/main/java/com/example/dkirilova/gym/fragments/ExerciseDetailsFragment.java

```diff
@@ -147,4 +150,18 @@ public void openDetails(Gym gym) {
             ((MainActivity) getActivity()).openGymDetailsFragment(gym);
         }
     }
+    @Override
+    public void setImage(ImageView imageView, String strUri) {
+        if (imageView == null || !Validator.isValidString(strUri)) {
+            return;
+        }
+        Uri uri = Uri.parse(strUri);
+
+        try {
+            Bitmap bitmap = MediaStore.Images.Media.getBitmap(getActivity().getContentResolver(), uri);
+            imageView.setImageBitmap(bitmap);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
log the error instead or maybe even show a toast to inform the user of the issue
printing the stacktrace is not that helpful on its own